From: Felix Fietkau <nbd@openwrt.org>
Date: Sat, 5 Dec 2015 13:53:36 +0100
Subject: [PATCH] net: remove unnecessary semicolon in
 netdev_alloc_pcpu_stats()

This semicolon causes a build error if the function call is wrapped in
parentheses.

Fixes: aabc92bbe3cf ("net: add __netdev_alloc_pcpu_stats() to indicate gfp flags")
Reported-by: Imre Kaloz <kaloz@openwrt.org>
Signed-off-by: Felix Fietkau <nbd@openwrt.org>
---

--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -2083,7 +2083,7 @@ struct pcpu_sw_netstats {
 })
 
 #define netdev_alloc_pcpu_stats(type)					\
-	__netdev_alloc_pcpu_stats(type, GFP_KERNEL);
+	__netdev_alloc_pcpu_stats(type, GFP_KERNEL)
 
 #include <linux/notifier.h>
 
